### PR TITLE
Clean up contributing formatting, add a Skia gold callout

### PR DESCRIPTION
**Pull request by matanlurey**
I am open to suggestions for other things to write under "Skia Gold".

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,15 +40,38 @@ with `gn format`.
...
 
-Tests will be executed on CI, but some tests will be executed before PR's can be merged (presubmit) and others after they have been merged (postsubmit). Ideally everything would be presubmit but tests that take up more resources are executed in postsubmit.
+Tests will be executed on CI, but some tests will be executed before PR's can be
+merged (presubmit) and others after they have been merged (postsubmit). Ideally
+everything would be presubmit but tests that take up more resources are executed
+in postsubmit.
+
+### Skia Gold
+
+The Flutter engine uses [Skia Gold][] for image comparison tests which fail if:
```
Comment (gaaclarke) [inline CONTRIBUTING.md:65]: This is a broken link methinks.

**Review by gaaclarke — APPROVED** [review]
LGTM.  Do you think it's worth noting the test harnesses that are feeding into skia gold?  (ie, impeller_golden_tests, scenario_app android, web)

File: CONTRIBUTING.md (hunk 1 above)
Comment (matanlurey) [reply CONTRIBUTING.md:65]: My understand is the implicit default is lowercased, but I can make it explicit.

Comment (gaaclarke) [reply CONTRIBUTING.md]: could be, it's hard to verify in githubs editor =T  thanks

**Pull request merged**